editor/header/preview-button/index.js

```diff
@@ -73,6 +73,16 @@ export class PreviewButton extends Component {
 			'about:blank',
 			this.getWindowTarget()
 		);
+
+		const popupLoader = renderToString(
+			<div>
+				<p>Please wait&hellip;</p>
+				<p>Generating preview.</p>
+			</div>
+		);
+		const css = '<style type="text/css"> div { margin-top: 25%; } p { text-align: center; } </style>';
+		this.previewWindow.document.write( css );
+		this.previewWindow.document.write( popupLoader );
```

Comment:
The docs for `document.write` encourage calling `document.close` after writing has completed:

>Once you have finished writing, it is recommended to call `document.close()` to tell the browser to finish loading the page. 

https://developer.mozilla.org/en-US/docs/Web/API/Document/write#Notes